openapi.yml

```diff
@@ -4941,6 +4977,54 @@ components:
                         colour: RED
                         last_name: some text
                         first_name: some text
+        NotificationsCreateParams:
+            description: ''
+            required:
+                - target
+                - message
+                - channels
+            type: object
+            properties:
+                target:
```

Comment:
This is to pass 'GUEST' and/or 'HOST', right?

Reply:
If this is a limited set of possible values and is only *one* value at a time, an enum would be ideal

Reply:
>If this is a limited set of possible values and is only one value at a time, an enum would be ideal

I was about to say exactly this :D Limit the values using an enum